src/components/actions/tests/ActionBarConnected.spec.tsx

```diff
@@ -1,35 +1,49 @@
-import { mount, ReactWrapper } from 'enzyme';
```

Comment:
Is the removal of the bracket spacing wanted?
I got this problem too when I was lintfixing 😄

Reply:
removing bracket spacing is where we want to go, I'll put some time in the near future to see how we can solve this through tsjs.